csharp/src/Apache.Arrow/C/CArrowArrayExporter.cs

```diff
@@ -190,26 +191,43 @@ private unsafe static void ConvertRecordBatch(ExportedAllocationOwner sharedOwne
 #endif
         private unsafe static void ReleaseArray(CArrowArray* cArray)
         {
-            Dispose(&cArray->private_data);
+            for (long i = 0; i < cArray->n_children; i++)
+            {
+                CArrowArray.CallReleaseFunc(cArray->children[i]);
+            }
+            if (cArray->dictionary != null) {
```

Comment:
While I have a personal preference for 1TBS, the rest of the code (and the default in VS) uses Allman-style braces and it would be nice to match. (Applies to a few other locations.)

Reply:
Ahah, sorry. Feel free to flag any other onces that I may have overlooked, as it's a bit difficult to find them all by staring.